test/image-blueprints/group4/rhel92-source-isolated.toml

```diff
@@ -38,6 +42,14 @@ enabled = ["mdns", "ssh", "http", "https"]
 name = "trusted"
 sources = ["10.42.0.0/16", "169.254.169.1"]
 
+[[customizations.files]]
+path = "/etc/sysconfig/qemu-ga"
+data = """
+# This is a systemd environment file, not a shell script.
+# It provides settings for "/lib/systemd/system/qemu-guest-agent.service".
+BLOCK_RPCS=
+FSFREEZE_HOOK_PATHNAME=/etc/qemu-ga/fsfreeze-hook"""
```

Comment:
Can we add to the comment the reason for setting these variables?

Reply:
+1

Reply:
I'm not sure the comment explains why we need those settings. Could we please elaborate a bit more?
The idea is for someone to look at the code and understand why these 2 are applied.

Reply:
Yes, the comments look good. I requested the update before it was added.